Sources/Fluent/Entity/Entity.swift

```diff
@@ -82,6 +84,10 @@ extension Entity {
     public static var name: String {
         return String(describing: self).lowercased()
     }
+    
+    public static var idType: Schema.Field.KeyType? {
```

Comment:
That's a good point. Since the entity has access to `database` could we just return `database.driver.idType` by default?